.eslintrc.js

```diff
@@ -4,29 +4,27 @@ module.exports = {
         es2021: true,
         mocha: true
     },
-    parser: '@babel/eslint-parser',
+    parser: '@typescript-eslint/parser',
     extends: [
-        // 'plugin:@typescript-eslint/recommended',
+        'plugin:@typescript-eslint/recommended'
         // 'plugin:import/recommended'``
     ],
-    // 'parser': '@typescript-eslint/parser',
     parserOptions: {
         ecmaVersion: 13,
         requireConfigFile: false
     },
     plugins: [
-        // '@typescript-eslint',
+        '@typescript-eslint',
         // 'import',
         'filenames',
         'prefer-arrow'
     ],
     ignorePatterns: [
         'node_modules',
         '*.min.js',
-        'example/*',
+        'src/example/*',
```

Comment:
Let's not move examples directory to `src`. All the examples are completely different projects which may be also written in `ts` (and some of them actually are) implying different `ts` configuration.

Reply:
scripts, resources, config and tests are not in src. This is probably a forgotten change.
Not sure about truffle and internal, it depends on how are they used. If we import some of these files anywhere, we can't move them out of src.